applications/q0/q0_utils.py

```diff
@@ -291,8 +290,7 @@ def gen_axis(title, xlabel, ylabel):
     return ax
 
 
-def redraw_axis(canvas, title, xlabel, ylabel):
-    # type: (FigureCanvasQTAgg, str, str, str) -> None
+def redraw_axis(canvas, title: FigureCanvasQTAgg, xlabel: str, ylabel: str):
```

Comment:
Did the canvas and title type annotations get swapped? Looks different from the old q0_utils.py file. Should it be:
`def redraw_axis(canvas: FigureCanvasQTAgg, title: str, xlabel: str, ylabel: str)`?